athena-dynamodb/src/test/java/com/amazonaws/athena/connectors/dynamodb/DynamoDbIntegTest.java

```diff
@@ -47,20 +53,87 @@ public class DynamoDbIntegTest extends IntegrationTestBase {
 
     private final String dynamodbDbName;
     private final String lambdaFunctionName;
-    private final String tableName;
-    private final DdbTableUtils ddbTableUtils;
+    private final String movieTableName;
+    private final DdbTableUtils tableUtils;
+
+    private static final String DDB_PARTITION_KEY = "title";
+    private static final String DDB_SORT_KEY = "year";
 
     public DynamoDbIntegTest()
     {
         Map<String, Object> userSettings = getUserSettings().orElseThrow(() ->
                 new RuntimeException("user_settings attribute must be provided in test-config.json."));
         dynamodbDbName = (String) userSettings.get("dynamodb_db_name");
         lambdaFunctionName = getLambdaFunctionName();
-        tableName = String.format("dynamodbit_%s", UUID.randomUUID().toString().replace('-', '_'));
-        ddbTableUtils = new DdbTableUtils(tableName);
+        movieTableName = String.format("dynamodbit_%s", UUID.randomUUID().toString().replace('-', '_'));
+        tableUtils = new DdbTableUtils();
+    }
+
+    /**
+     * Insert rows into the newly created DDB table.
+     */
+    protected void addMovieItems()
+    {
+        tableUtils.putItem(movieTableName, createMovieItem("2014", "Interstellar", "Christopher Nolan",
+                ImmutableList.of("Matthew McConaughey", "John Lithgow", "Ann Hathaway",
```

Comment:
If we end up preferring the Java standard library over Guava, replace this with: `List.of()`

https://docs.oracle.com/javase/9/docs/api/java/util/List.html#immutable

Reply:
We use Guava everywhere here, so I kept that pattern.